extras/AFC_BoxTurtle.py

```diff
@@ -26,70 +26,69 @@ def handle_connect(self):
         self.logo_error+='R |          |\ <span class=secondary--text>X</span> |\n'
         self.logo_error+='! \_________/ |___|</error>\n'
 
-    def system_Test(self, UNIT, LANE):
+    def system_Test(self, UNIT, LANE, delay):
         msg = ''
         CUR_LANE = self.printer.lookup_object('AFC_stepper ' + LANE)
         try: CUR_EXTRUDER = self.printer.lookup_object('AFC_extruder ' + CUR_LANE.extruder_name)
         except:
-           error_string = 'Error: No config found for extruder: ' + CUR_LANE.extruder_name + ' in [AFC_stepper ' + CUR_LANE.name + ']. Please make sure [AFC_extruder ' + CUR_LANE.extruder_name + '] config exists in AFC_Hardware.cfg'
-           self.AFC.AFC_error(error_string, False)
-           return False
+            error_string = 'Error: No config found for extruder: ' + CUR_LANE.extruder_name + ' in [AFC_stepper ' + CUR_LANE.name + ']. Please make sure [AFC_extruder ' + CUR_LANE.extruder_name + '] config exists in AFC_Hardware.cfg'
+            self.AFC.AFC_error(error_string, False)
+            return False
 
-        if CUR_EXTRUDER.buffer_name !=None:
-            CUR_EXTRUDER.buffer = self.printer.lookup_object('AFC_buffer ' + CUR_EXTRUDER.buffer_name)
-            # Run test reverse/forward on each lane
-            CUR_LANE.extruder_stepper.sync_to_extruder(None)
-            CUR_LANE.move( 5, self.AFC.short_moves_speed, self.AFC.short_moves_accel, True)
-            self.AFC.reactor.pause(self.AFC.reactor.monotonic() + 1)
-            CUR_LANE.move( -5, self.AFC.short_moves_speed, self.AFC.short_moves_accel, True)
+        # Run test reverse/forward on each lane
```

Comment:
Looks like alot of changes, but its just removing the if check and moving the code over to the left by one tab